contracts/Elections.sol

```diff
@@ -86,16 +83,16 @@ contract Elections is IElections, ContractRegistryAccessor, WithClaimableFunctio
 		getCommitteeContract().memberComplianceChange(addr, isCompliant);
 	}
 
-	function notifyReadyForCommittee() external onlyNotBanned {
-		address sender = getMainAddrFromOrbsAddr(msg.sender);
+	function notifyReadyForCommittee() external onlyNotBanned { // TODO onlyNotBanned doesn't check etehreum address
+		address sender = getMainAddrFromOrbsAddr(msg.sender); // This validates the validator is registered
```

Comment:
rename sender to guardianAddress / senderAddress

Reply:
Already done, in another open PR